ironfish/src/wallet/account.ts

```diff
@@ -447,7 +456,7 @@ export class Account {
   }> {
     let unconfirmedCount = 0
 
-    const unconfirmed = await this.getUnconfirmedBalance(assetIdentifier, tx)
+    const { balance: unconfirmed } = await this.getUnconfirmedBalance(assetIdentifier, tx)
```

Comment:
makes me think we should just have this as unconfirmed so its clear when looking at the model itself its unconfirmed

Reply:
Agreed, changed it to unconfirmed!